src/openrct2-ui/audio/AudioMixer.cpp

```diff
@@ -312,15 +312,16 @@ namespace OpenRCT2::Audio
             // Apply effects
             if (rate != 1)
             {
-                int32_t inRate = (int32_t)(bufferLen / byteRate);
+                int32_t inRate = static_cast<int32_t>((bufferLen / byteRate));
                 int32_t outRate = numSamples;
                 if (bytesRead != readLength)
                 {
                     inRate = _format.freq;
                     outRate = _format.freq * (1 / rate);
                 }
                 _effectBuffer.resize(length);
-                bufferLen = ApplyResample(channel, buffer, (int32_t)(bufferLen / byteRate), numSamples, inRate, outRate);
+                bufferLen = ApplyResample(
+                    channel, buffer, static_cast<int32_t>((bufferLen / byteRate)), numSamples, inRate, outRate);
```

Comment:
```suggestion
                    channel, buffer, static_cast<int32_t>(bufferLen / byteRate), numSamples, inRate, outRate);
```

OK, I have done enough of these line notes. Could you remove the double parentheses in the rest of the diff?